cluster-up/scale.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/bash -x
+
+# Add Rhel 7.7 Worker
+
+CUSTOM_IMAGE=${CUSTOM_IMAGE:-"0"}
+PROVISION_MODE=${PROVISION_MODE:-"0"}
+REPO_FILE=${REPO_FILE:-"0"}
+
+BASE_IMAGE="rhel-server-7.7-x86_64-kvm.qcow2"
+export KUBECONFIG=/root/install/auth/kubeconfig
+
+SELF_PID=$(echo $$)
+ps -ef | grep scale.sh | grep -v grep | awk '{print $2}' | grep -v $SELF_PID | xargs kill
+pkill dnf
+pkill ansible-playbook
+
+dnf install -y libguestfs-tools
+if [ ! -f /root/.ssh/id_rsa.pub ]
+then
+   echo "id_rsa tuple must exists, Generating"
+   ssh-keygen -t rsa -f /root/.ssh/id_rsa -N ''
+fi
+
+dnf install -y https://repo.fedora.md/centos/7/virt/x86_64/ovirt-4.2/common/ansible-2.8.2-1.el7.noarch.rpm
+dnf install -y openshift-ansible openshift-clients jq
+
+NETLIST=$(virsh net-list | grep -v def | grep active | awk '{print $1}')
+
+cd /tmp
+if [ $CUSTOM_IMAGE = "0" ] || [ $PROVISION_MODE = "1" ]; then
+   qemu-img create -f qcow2 el77-worker-1.img 20G
```

Comment:
can we rename el77-worker-1.img to just worker-1.img so we remove more references to rhel-7 ?